core/src/main/java/org/apache/accumulo/core/client/mapreduce/InputFormatBase.java

```diff
@@ -323,7 +382,7 @@ protected static TabletLocator getTabletLocator(JobContext context) throws Table
      * @since 1.6.0
      */
     @Override
-    protected void setupIterators(TaskAttemptContext context, Scanner scanner, String tableName, org.apache.accumulo.core.client.mapreduce.RangeInputSplit split) {
+    protected void setupIterators(TaskAttemptContext context, ScannerBase scanner, String tableName, org.apache.accumulo.core.client.mapreduce.AccumuloInputSplit split) {
```

Comment:
Sorry, we can't change these as they'd break API from 1.6.0. We can add to the API for 1.7.0, but we can't remove. Need to deprecate the ones that take Scanner and add in a new one for ScannerBase.